snackbar/snackbar/src/androidTest/java/com/microsoft/device/dualscreen/snackbar/SnackbarContainerPortraitTests.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ */
+
+package com.microsoft.device.dualscreen.snackbar
+
+import androidx.test.filters.MediumTest
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.microsoft.device.dualscreen.snackbar.SnackbarPosition.BOTH
+import com.microsoft.device.dualscreen.snackbar.SnackbarPosition.END
+import com.microsoft.device.dualscreen.snackbar.SnackbarPosition.START
+import com.microsoft.device.dualscreen.snackbar.test.R
+import com.microsoft.device.dualscreen.testing.getDeviceModel
+import com.microsoft.device.dualscreen.testing.spanFromStart
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@MediumTest
+@RunWith(AndroidJUnit4ClassRunner::class)
+class SnackbarContainerPortraitTests : SnackbarContainerTests() {
+
+    @Test
+    fun testPositionOnSingleScreen() {
+        val margin = SnackbarContainer.COORDINATOR_LAYOUT_MARGIN
+        testMargins(TestParams(R.string.snackbar_to_start, START, margin, 0, margin, margin))
+        testMargins(TestParams(R.string.snackbar_to_end, END, margin, 0, margin, margin))
+        testMargins(TestParams(R.string.snackbar_to_both, BOTH, margin, 0, margin, margin))
+    }
+
+    @Test
+    fun testPositionOnDualScreen() {
+        windowLayoutInfoConsumer.reset()
+        uiDevice.spanFromStart()
+        windowLayoutInfoConsumer.waitForWindowInfoLayoutChanges()
+
+        val margin = SnackbarContainer.COORDINATOR_LAYOUT_MARGIN
+        val rightMargin = with(uiDevice.getDeviceModel()) {
+            paneWidth - margin
+        }
+        testMargins(TestParams(R.string.snackbar_to_start, START, margin, 0, rightMargin, margin))
+        val leftMargin = with(uiDevice.getDeviceModel()) {
+            paneWidth + +foldWidth + margin
```

Comment:
Is this a typo or just a pre-increment or another thing? maybe better to enclose it into parenthesis to avoid misunderstandings.

Reply:
Is just a type. I'll remove the + sign. Thanks

Reply:
Done